sui/wormhole/README.md

```diff
@@ -23,3 +23,10 @@ TODO: up to date notes on where and how we use dynamic fields.
 
 Sui currently does not have fine-grained timestamps, so we use
 `tx_context::epoch(ctx)` in place of on-chain time in seconds.
+
+# Source verification
+
+Verify that the Move sources here match the package published on-chain by running
+`sui client verify-source --verify-deps`
+
+Make sure that `sui client active-env` is connected to a Sui testnet RPC.
```

Comment:
Is this only for testnet and not mainnet?

Reply:
No, the procedure is general whether you're verifying mainnet or testnet code. But ofc, if one checked-out the testnet branch, a testnet RPC should be used. I may update the description here to be more general